src/components/organisms/TimingEvent/TimingEvent.tsx

```diff
@@ -1,80 +1,114 @@
 import { format } from "date-fns";
 
+import { SpaceWithId } from "types/id";
 import { WorldEvent } from "types/venues";
 
 import { eventEndTime, eventStartTime } from "utils/event";
 import { formatTimeLocalised } from "utils/time";
 
-import { useRelatedVenues } from "hooks/useRelatedVenues";
+import { useShowHide } from "hooks/useShowHide";
+
+import { TimingDeleteModal } from "../TimingDeleteModal";
+import { TimingEventModal } from "../TimingEventModal";
 
 export type TimingEventProps = {
   event: WorldEvent;
-  setShowCreateEventModal: () => void;
-  setEditedEvent: (event: WorldEvent) => void;
+  space?: SpaceWithId;
 };
 
-export const TimingEvent: React.FC<TimingEventProps> = ({
-  event,
-  setShowCreateEventModal,
-  setEditedEvent,
-}) => {
-  const [display, setDisplay] = useState(false);
-
-  const { findVenueInRelatedVenues } = useRelatedVenues();
-  const space = findVenueInRelatedVenues({ spaceId: event.spaceId });
+export const TimingEvent: React.FC<TimingEventProps> = ({ event, space }) => {
+  const {
+    isShown: showDeleteEventModal,
+    show: setShowDeleteEventModal,
+    hide: setHideDeleteEventModal,
+  } = useShowHide();
 
-  const showButton = (e: React.MouseEvent) => {
-    e.preventDefault();
-    setDisplay(true);
-  };
-  const hideButton = (e: React.MouseEvent) => {
-    e.preventDefault();
-    setDisplay(false);
-  };
+  const {
+    isShown: isShownCreateEventModal,
+    show: showCreateEventModal,
+    hide: hideCreateEventModal,
+  } = useShowHide();
 
   return (
-    <div
-      key={event.id}
-      className="TimingEvent"
-      onMouseEnter={showButton}
-      onMouseLeave={hideButton}
-    >
-      <div className="TimingEvent__time">
-        <p>{format(eventStartTime({ event }), "do MMM")}</p>
-        <p className="TimingEvent__time-start">
-          {formatTimeLocalised(eventStartTime({ event }))}
-        </p>
-        <p>{formatTimeLocalised(eventEndTime({ event }))}</p>
-      </div>
-      <div className="TimingEvent__details">
-        <p>
-          <span className="TimingEvent__details-name">{event.name}</span> by{" "}
-          <span className="TimingEvent__details-host">{event.host}</span>
-        </p>
-        <p className="TimingEvent__details-description">{event.description}</p>
-        <p className="TimingEvent__details-room">
-          in <span className="event-details-room-name">{space?.name}</span>
-        </p>
-      </div>
+    <>
+      <div key={event.id} className="">
+        <div className="bg-white divide-y divide-gray-200">
+          <div className="flex justify-start items-start">
+            <div className="px-6 py-4 w-96">
+              <div className="text-sm font-medium text-gray-900">
+                {format(eventStartTime({ event }), "do MMM")}
+              </div>
+              <div className="text-sm text-gray-500">
+                {formatTimeLocalised(eventStartTime({ event }))} -{" "}
```

Comment:
To disambiguate from unintended IDE addition of `{" "}`.

```suggestion
                {formatTimeLocalised(eventStartTime({ event }))} -{STRING_SPACE}
```